src/grafana_dashboards/vault.json

```diff
@@ -858,6 +857,95 @@
             ],
             "title": "Allocated Bytes",
             "type": "stat"
+        },
+        {
+            "collapsed": false,
+            "gridPos": {
+                "h": 1,
+                "w": 24,
+                "x": 0,
+                "y": 28
+            },
+            "id": 96,
+            "panels": [],
+            "title": "Audit Logs",
+            "type": "row"
+        },
+        {
+            "datasource": {
+                "type": "loki",
+                "uid": "PCFE195BA03E127CE"
+            },
+            "description": "",
+            "gridPos": {
+                "h": 8,
+                "w": 11,
+                "x": 0,
+                "y": 29
+            },
+            "id": 93,
+            "links": [],
+            "options": {
+                "dedupStrategy": "none",
+                "enableLogDetails": true,
+                "prettifyLogMessage": false,
+                "showCommonLabels": false,
+                "showLabels": false,
+                "showTime": true,
+                "sortOrder": "Descending",
+                "wrapLogMessage": false
+            },
+            "targets": [
+                {
+                    "datasource": {
+                        "type": "loki",
+                        "uid": "PCFE195BA03E127CE"
+                    },
+                    "editorMode": "code",
+                    "expr": "{charm=\"vault-k8s\"} | json | type = `response` or type = `request`",
+                    "queryType": "range",
+                    "refId": "A"
+                }
+            ],
+            "title": "All Vault Logs",
+            "type": "logs"
+        },
+        {
+            "datasource": {
+                "type": "loki",
+                "uid": "PCFE195BA03E127CE"
+            },
+            "gridPos": {
+                "h": 8,
+                "w": 13,
+                "x": 11,
+                "y": 29
+            },
+            "id": 95,
+            "options": {
+                "dedupStrategy": "none",
+                "enableLogDetails": true,
+                "prettifyLogMessage": false,
+                "showCommonLabels": false,
+                "showLabels": false,
+                "showTime": false,
+                "sortOrder": "Descending",
+                "wrapLogMessage": false
+            },
+            "targets": [
+                {
+                    "datasource": {
+                        "type": "loki",
+                        "uid": "PCFE195BA03E127CE"
```

Comment:
Similar comment, we shouldn't be explicit about the loki uid. Please look at other panels here.

Reply:
Updated